UniversalScanner/packages.config

```diff
@@ -0,0 +1,6 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<packages>

```

Comment:
Please remove packages, I tend to avoid packages to reduce use of external library.
Also, please note the version of dotNet I use is different.